src/utils/fees.ts

```diff
@@ -46,46 +60,132 @@ function isEIP1559Fees(chain: ChainId) {
   return chain === ChainId.Ethereum || chain === ChainId.Polygon || chain === ChainId.Avalanche;
 }
 
+/*
+ * Probable fee for EIP1559 (BASE + TIP)
+ */
+function probableFeePerUnitEIP1559(suggestedGasFee: EIP1559Fee) {
+  return suggestedGasFee.suggestedBaseFeePerGas || 0 + suggestedGasFee.maxPriorityFeePerGas;
+}
 
-async function getSendAmountFee(accountId: AccountId, asset: Asset, amount?: BN) {
-  const getMax = amount === undefined
-  const sendFees: {
-    [speed in keyof FeeDetails]: BN
-  } = { slow: new BN(0), average: new BN(0), fast: new BN(0), }
+/*
+ * Max fee for EIP1559
```

Comment:
Nice, thanks for breaking everything up into smaller functions that you can easily call and display the values :)